config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -71,10 +71,10 @@ periodics:
       - --test-cmd-args=--provider=kubemark
       - --test-cmd-args=--report-dir=/workspace/_artifacts
       - --test-cmd-args=--testconfig=testing/load/config.yaml
+      - --test-cmd-args=--testoverrides=./testing/density/5000_nodes/override.yaml
```

Comment:
This one is incorrect - we're moving them out from density, because overrides are actually per the whole job, not per test.
The one passed currently seems correct:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/overrides/5000_nodes.yaml

Reply:
Problem is different - golang tests are not run from HEAD

Reply:
Ahh - I see.

Can you maybe add a comment for these, sth like:
Change the override to ... once golang tests will migrate to 1.19+ branch?

Reply:
Done

Reply:
Actually, it's just got fixed in https://github.com/kubernetes/test-infra/pull/17753, please wait till it gets merged and then most likely revert this change

Reply:
Reverted